source/com.microsoft.tfs.client.clc/src/com/microsoft/tfs/client/clc/messages.properties

```diff
@@ -77,7 +77,10 @@ Command.CommandDoesNotSupportOptionFormat=Command ''{0}'' does not support optio
 Command.CouldNotDetermineCollectionURL=The Team Foundation Server project collection URL to use could not be determined by the options or argument paths provided.  Specify a workspace, or use the 'collection' option.
 Command.DefaultCredsUnavailableLibraryLoadFailure=Default credentials are unavailable because the system libraries could not be loaded.  Check the program log files for details.
 Command.DefaultCredsUnavailableNoTicket=Default credentials are unavailable because no Kerberos ticket or other authentication token is available.
-Command.InteractiveAuthenticationFailed=Failed to retrieve credentials interactively.  Check the program log files for details. Interactive authentication is only supported on Oracle Java SE 7 update 6 or higher, Oracle Java SE 8, or OpenJDK 8 runtime.
+Command.InteractiveAuthenticationFailed=Failed to retrieve credentials interactively.  Check the program log files for details. To bypass web browser based interactive authentication, please set the {0} environment variable to true. 
+Command.InteractiveAuthenticationFailedDetailedLog1=Failed to authenticate interactively with web browser. This requires either JavaFX or SWT based web browser control:
+Command.InteractiveAuthenticationFailedDetailedLog2=1. JavaFX web browser control is only supported on Oracle Java SE 7 update 6 or higher, Oracle Java SE 8, or OpenJDK 8 runtime (Please note you may need to compile OpenJFX project yourself).
+Command.InteractiveAuthenticationFailedDetailedLog3=2. To launch SWT browser with a specific XULRunner, please set the value of Java system property 'org.eclipse.swt.browser.XULRunnerPath' to the full path of the specific XULRunner.
```

Comment:
At first I didn't localize those strings precisely because they would end up in logs.  Then I thought maybe some users would read logs and they could figure out the problem themselves.  It doesn't hurt to translate those strings.